.clj-kondo/config.edn

```diff
@@ -44,6 +44,10 @@
     (clojure.core.logic/run)
     (clojure.test/is [query= =? malli=])
     (clojure.tools.macro/macrolet)
+    (etaoin.api/with-chrome)
```

Comment:
instead of adding a bunch of things like this in here we should write Kondo config hooks so it can properly lint these forms